app/services/template_versioning_service.rb

```diff
@@ -92,8 +92,6 @@ def self.force_publish_now!(requirement_template)
 
     ModelCallbackJob.perform_async(template_version.class.name, template_version.id, "force_publish_now!")
 
-    template_version = publish_version!(template_version, true)
```

Comment:
@taw27 Hoping you could help me understand why this line was removed

Reply:
@bkmaibach see previous line. This method is backgrounded already, but I forgot to remove the synchronous line before ` template_version = publish_version!(template_version, true)` when I did the background job feature